cookbooks/view_pings_cep.md

```diff
@@ -0,0 +1,57 @@
+# See My Pings
+
+So you want to see what you're sending the telemetry pipeline, huh? Well follow these steps and we'll have you reading some JSON in no time.
+
+Note that some of these steps were stolen from the [Creating a Real-Time Analysis Plugin Cookbook](realtime_analysis_plugin.md).
+
+## Steps to Create a Viewing Output
+
+1. Get your client_id from whatever product you're using. For desktop, it's available in `about:telemetry`.
+
+2. Go to the CEP site: https://pipeline-cep.prod.mozaws.net/
+
+3. Login/Register using your Google `@mozilla.com` account
+
+4. Click on the `Analysis Plugin Deployment` tab
```

Comment:
There's a separate "Matcher" tab now! It's much easier to use in the case where you just want to look at a few pings. Can you add an intro section that shows you how to simply examine some data using that tab, then takes you through capturing your own with an analysis plugin?

If that's beyond the scope here, let's file an issue for it.

Reply:
Yes, I think this cookbook should be one thing: I want to see my own pings. You can't do that in the matcher. That should live somewhere else about the CEP.